packages/api-cardano-db-hasura/test/__snapshots__/transactions.query.test.ts.snap

```diff
@@ -254,8 +415,12 @@ Object {
           },
         },
       },
+      "redeemers": Array [],
+      "scriptSize": 0,
+      "scripts": Array [],
       "size": 220,
       "totalOutput": "4924799478660",
+      "validContract": true,
```

Comment:
@rhyslbw what does this field mean? And should it be true value here?

Reply:
![image](https://user-images.githubusercontent.com/303881/129024742-3919878e-af42-40dc-9a47-db660cc3d81e.png)

https://github.com/input-output-hk/cardano-db-sync/blob/alonzo-purple-1.0/doc/schema.md#tx

Reply:
Thank you